src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -176,6 +184,11 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                .WithDescription("If provided, a path to the file that will be used to write the output when using --enable-json-output.")
                .Callback(jsonOutputFile => JsonOutputFile = jsonOutputFile);
 
+            Parser
+               .Setup<string>("runtime")
+               .WithDescription("If provided, determines which version of the host to start. Allowed values are inproc6, inproc8, and default.")
```

Comment:
nit: use consts for allowed values (noticed InProc8HostRuntime and 6 defined above)